benchmarks/viscoelastic_flexure_finite-length_elastic_plate/viscoelastic_flexure_finite-length_elastic_plate.prm

```diff
@@ -0,0 +1,180 @@
+# This benchmark examines flexure of a finite-length elastic plate based
```

Comment:
Can we maybe rename this benchmark to something shorter and still explanatory, like `viscoelastic_plate_flexure`? It would be easier to read and understand.